### DISCUSSION TITLE
Using matchDepNames vs matchPackageNames, is one deprecated?

### DISCUSSION THREAD

**Discussion by ryancausey** [renovatebot/renovate#24623] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

ghcr.io/renovatebot/renovate:36.105.0@sha256:8940e322a96fbc54b42c633ba65a61ab7c7ea526a226bcfe30ac4a40e67aeee3

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

Nothing is broken yet. It's simply a log message I noticed.

### Wanted end result.

Clarity on which to use, `matchDepNames` or `matchPackageNames`. The INFO log output indicates I should use `matchDepNames` and that `matchPackageNames` is deprecated, but the documentation doesn't indicate that.

### What you tried so far.

I checked the docs for both:

* https://docs.renovatebot.com/configuration-options/#matchpackagenames
* https://docs.renovatebot.com/configuration-options/#matchdepnames

However, the `matchPackageNames` docs are more fleshed out and make it look like the correct choice. The `matchDepNames` docs simply say "For advanced use only! Use at your own risk!".

Is there any difference between the two? Is `matchPackageNames` deprecated in favor of `matchDepNames`?

### Relevant debug logs

<details><summary>Logs</summary>

```
INFO: Use matchDepNames instead of matchPackageNames (repository=munipal/alarm-tracking-services/munipal-graphql-api)
```

</details>

**Comment by rarkins** [comment, marked as answer]
Neither is deprecated.

`matchDepNames` matches only the `depName` value for a dependency.
`matchPackageNames` tries matching the `packageName` value but then falls back to matching `depName` and warns if it matches. It is the fallback to `depName` within `matchPackageNames` which is deprecated.

There should be metadata below the `INFO` line you pasted
